frontend/src/components/ConcResultPlot.tsx

```diff
@@ -1,19 +1,35 @@
 import React from "react";
-import { Typography } from "@equinor/eds-core-react";
 import { SimulationResults } from "../dto/SimulationResults";
-import Plot from "react-plotly.js";
-
+import { Bar } from "react-chartjs-2";
+import { Chart as ChartJS, CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend } from "chart.js";
 import { extractPlotData } from "../functions/Formatting";
 
+ChartJS.register(CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend);
+
 interface ResultsProps {
     simulationResults: SimulationResults;
 }
 
 const ResultConcPlot: React.FC<ResultsProps> = ({ simulationResults }) => {
+    const chartData = extractPlotData(simulationResults);
+    const options = {
+        responsive: true,
+        scales: {
+            x: {
+                ticks: {
+                    autoSkip: false,
+                },
+            },
+        },
+    };
+
     return (
         <>
-            <Typography variant="h4">Change in concentrations</Typography>
-            <Plot data={extractPlotData(simulationResults)} layout={{ title: { text: "" } }} />
+            <div style={{ width: "600px", marginLeft: "0" }}>
+                <Bar data={chartData} options={options} />
+            </div>
+            <br />
+            <br />
```

Comment:
Perhaps a marginBottom in the style instead of the double `<br />`? :)

Reply:
I suggest using `em` units, which corresponds to 1 font height. So, `marginBottom: "2em"`